CHANGELOG.md

```diff
@@ -1,250 +1,331 @@
+# 0.22.1
+
```

Comment:
Why did we add a space after the version number? This doesn't seem to be necessary.